lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerPlatform.cpp

```diff
@@ -318,75 +231,67 @@ GDBRemoteCommunicationServerPlatform::Handle_qKillSpawnedProcess(
 
   lldb::pid_t pid = packet.GetU64(LLDB_INVALID_PROCESS_ID);
 
-  // verify that we know anything about this pid. Scope for locker
-  {
-    std::lock_guard<std::recursive_mutex> guard(m_spawned_pids_mutex);
-    if (m_spawned_pids.find(pid) == m_spawned_pids.end()) {
-      // not a pid we know about
-      return SendErrorResponse(10);
-    }
+  // verify that we know anything about this pid.
+  if (SpawnedProcessFinished(pid)) {
+    // not a pid we know about
+    return SendErrorResponse(10); // ECHILD
```

Comment:
Let's get rid of these comments. If we really wanted these to have a meaning, they should be one of the GDB_ESOMETHING constants defined in GDBRemoteCommunication.h.

Reply:
Done.